lib/api.rb

```diff
@@ -110,11 +118,11 @@ def validation_error(msg)
 
       u = User.find_by_email(params[:username])
       if !u
-        return validation_error("Invalid username")
+        return validation_error("Invalid username or password")
       end
 
       if !u.has_password_hash?(params[:password])
-        return validation_error("Invalid password")
+        return validation_error("Invalid password or password")
```

Comment:
I think you meant to have that say "Invalid username or password"

Reply:
Yes. If someone try to "guess" the username or password, we shouldn't say which one is wrong.

Reply:
@Pschittt I think the problem is in the wording - you wrote "Invalid password or password", while you meant "Invalid username or password". Can you please amend?

Reply:
Indeed. Forgot to change it. Done with the last commit.